test/new-e2e/tests/windows/iis.go

```diff
@@ -0,0 +1,85 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package windows contains helpers for Windows E2E tests
+package windows
+
+import (
+	"fmt"
+	"path"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+)
+
+// IISSiteDefinition represents an IIS site definition
+type IISSiteDefinition struct {
+	Name        string //  name of the site
+	BindingPort string // port to bind to, of the form '*:8081'
+	AssetsDir   string // directory to copy for assets
+}
+
+// InstallIIS installs IIS on the target machine
+func InstallIIS(host *components.RemoteHost) error {
+	script := `
+	function ExitWithCode($exitcode) {
+		$host.SetShouldExit($exitcode)
+		exit $exitcode
+	  }
+	$result = install-windowsfeature -name Web-Server -IncludeManagementTools
+	if (! $result.Success ) {
+		exit -1
+	}
+	if ($result.RestartNeeded -eq "Yes") {
+		ExitWithCode(3010)
+	}
+	`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
You could [embed](https://pkg.go.dev/embed) this script, saving it in a `[fixture|assets]/install-iis.ps1` file and import it to `[]byte` with 
```go
// go:embed [fixture|assets]/install-iis.ps1
var installScript []byte
```